doc/conf.py

```diff
@@ -25,14 +25,7 @@
 rtd = os.environ.get('READTHEDOCS') == 'True'
 if rtd:
     subprocess.check_call(['doxygen'])
-    # ReadTheDocs can't build the extension (no Boost), so we have to mock it
-    # See http://docs.readthedocs.io/en/latest/faq.html#i-get-import-errors-on-libraries-that-depend-on-c-modules
-    import unittest.mock
-    MOCK_MODULES = ['spead2._spead2', 'spead2._spead2.send', 'spead2._spead2.recv']
-    sys.modules.update((mod_name, unittest.mock.Mock()) for mod_name in MOCK_MODULES)
-    # Mocking certain classes causes subclasses not to be documented properly
-    sys.modules['spead2._spead2.send'].UdpStreamAsyncio = object
-    sys.modules['spead2._spead2.send'].UdpIbvStreamAsyncio = object
+    autodoc_mock_imports = ['spead2._spead2', 'spead2._spead2.send', 'spead2._spead2.recv']
```

Comment:
According to the docs, it's not necessary to list submodules, so I believe this can be reduced to:
```suggestion
    autodoc_mock_imports = ['spead2._spead2']
```

Reply:
Ahhh true! I knew this, but did the change in the hurry and didn't stop to think about it, good catch.

I also learned about autodoc's mock mechanism after doing my own mocking for some time. I've run into some small gotchas, but in almost all cases it does a pretty good work.

Reply:
I've applied the requested change